modules/web/js/ballerina/configs/designer-defaults.js

```diff
@@ -263,4 +264,11 @@ export const panelHeading = {
         heading: 14,
         padding: 3
     }
-};
\ No newline at end of file
+};
+
+export const timeout = {
```

Comment:
generic values are already defined in blockStatement, this values are specific to the the timeout, because work "Timeout" is longer.